target/linux/x86/base-files/etc/board.d/02_network

```diff
@@ -23,9 +23,12 @@ cisco-mx100-hw)
 	ucidef_set_network_device_path "eth11" "pci0000:00/0000:00:01.1/0000:02:00.2"
 	ucidef_set_interfaces_lan_wan "mgmt eth2 eth3 eth4 eth5 eth6 eth7 eth8 eth9 eth10 eth11" "wan"
 	;;
-pc-engines-apu1|pc-engines-apu2|pc-engines-apu3)
+pc-engines-apu1|pc-engines-apu2|pc-engines-apu3|pc-engines-apu5)
```

Comment:
APU5? Any link?

Reply:
Alas the APUv5 is a bespoke design for a channel partner, so there’s no public page for it.